src/test/kotlin/com/terraformation/backend/funder/db/FundingEntityStoreTest.kt

```diff
@@ -47,7 +49,7 @@ class FundingEntityStoreTest : DatabaseTest(), RunsAsUser {
 
   @Test
   fun `fetchOneById retrieves entity with projects`() {
-    insertOrganization()
+    organizationId // ensures that organization exists prior to projects
```

Comment:
An alternate way to do this would be to do `insertProject(name = "whatever", organizationId = organizationId)`.